src/main/java/me/zeroeightsix/kami/util/CoordUtil.java

```diff
@@ -0,0 +1,104 @@
+package me.zeroeightsix.kami.util;
+
+import net.minecraft.client.Minecraft;
+
+import me.zeroeightsix.kami.util.Coord;
+import java.io.*;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Scanner;
+
+import static me.zeroeightsix.kami.util.MessageSendHelper.sendChatMessage;
+
+/**
+ * @author dominikaaaa
```

Comment:
According to the git patch, you wrote the entirety of this file? So why are you giving other people credit?

Reply:
Actually, @cookiedragon234 I wrote the original file. They renamed it so it counts as a new file on Github.

Reply:
It says author dominikaaa?

Reply:
Oh, sorry, misread, I renamed the file to CoordUtil because LogUtil did not make much sense to me.